pallets/parachain-staking/src/weights.rs

```diff
@@ -184,41 +181,43 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 	/// Proof: System Account (max_values: None, max_size: Some(116), added: 2591, mode: MaxEncodedLen)
 	/// Storage: Balances Locks (r:1 w:1)
 	/// Proof: Balances Locks (max_values: None, max_size: Some(1287), added: 3762, mode: MaxEncodedLen)
+	/// Storage: Balances Freezes (r:1 w:0)
+	/// Proof: Balances Freezes (max_values: None, max_size: Some(37), added: 2512, mode: MaxEncodedLen)
 	/// Storage: ParachainStaking Total (r:1 w:1)
 	/// Proof Skipped: ParachainStaking Total (max_values: Some(1), max_size: None, mode: Measured)
 	/// Storage: ParachainStaking TopDelegations (r:0 w:1)
 	/// Proof Skipped: ParachainStaking TopDelegations (max_values: None, max_size: None, mode: Measured)
 	/// Storage: ParachainStaking BottomDelegations (r:0 w:1)
 	/// Proof Skipped: ParachainStaking BottomDelegations (max_values: None, max_size: None, mode: Measured)
-	/// The range of component `x` is `[3, 1000]`.
+	/// The range of component `x` is `[3, 200]`.
 	fn join_candidates(x: u32, ) -> Weight {
 		// Proof Size summary in bytes:
-		//  Measured:  `1807 + x * (37 ±0)`
-		//  Estimated: `26879 + x * (228 ±0)`
-		// Minimum execution time: 70_391_000 picoseconds.
-		Weight::from_parts(70_805_524, 26879)
-			// Standard Error: 8_036
-			.saturating_add(Weight::from_parts(84_366, 0).saturating_mul(x.into()))
-			.saturating_add(T::DbWeight::get().reads(6_u64))
+		//  Measured:  `1421 + x * (38 ±0)`
+		//  Estimated: `4752 + x * (41 ±0)`
+		// Minimum execution time: 75_412_000 picoseconds.
+		Weight::from_parts(85_543_270, 4752)
+			// Standard Error: 2_633
+			.saturating_add(Weight::from_parts(213_586, 0).saturating_mul(x.into()))
+			.saturating_add(T::DbWeight::get().reads(7_u64))
```

Comment:
This is a pretty big increase, is it expected?

Reply:
Oh, did these staking ones change because they're generated by a proper runtime instead of a mock runtime?

Reply:
...actually, why is staking the only one where we touched the pallet-level weights? They went up for the most part, which is probably "safe." It's used by other projects which may not generate their own weights, so maybe that's a good thing :shrug:

Reply:
The weights of some pallets were relying on substrate part that were not correctly configured.
I think the goal is to get default values for external team, but still generate weights for our runtimes once ready

Reply:
So yes we want to provide weight by default on pallets, and then a weight for our runtimes